### PR TITLE
Module 1: "Hello Microverse" project

### PR DESCRIPTION
**Pull request by svitalis123**
## I have setup Hello Microverse project
In this project:  
- I created a Hello world repository 
-   I created a GitHub branch

### CODE REVIEW

File: app.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: red;
+}
```
Comment (1GORDON) [inline app.css:3]: Great job @svitalis123 🤝

- [ ] [Optional] Kindly use a class instead of the `h1` to add style to the Hello-Microverse text.

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @svitalis123 ✋



Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

## Optional suggestions

##### _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

##### Cheers and Happy coding!:clap::clap::clap:
##### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: app.css (hunk 1 above)
Comment (svitalis123) [reply app.css:3]: Noted sir thank you

**Comment by svitalis123** [comment]
Thank you sir i will use class next time.    On Tuesday, August 16, 2022 at 10:59:37 PM GMT+3, Gordon Tinyefuza ***@***.***> wrote:  
 
 


@1GORDON approved this pull request.

Status: APPROVED 🟢
 
Hi @svitalis123 ✋
 


Your project is complete! There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 👏 👏 💯
 
Highlights!
   
   - Followed Gitflow ✔️
   - Good commit messages ✔️
   - Linters are passing ✔️
   - Descriptive Pull request ✔️
   - Professional readme ✔️

Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding! 👏 👏 👏
 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

_As described in the Code reviews limits policy you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this form.

In app.css:
> +h1 {
+  color: red;
+}

Great job @svitalis123 🤝
    
   -  [Optional] Kindly use a class instead of the h1 to add style to the Hello-Microverse text.

**Pull request merged**